app/presenters/activity_csv_presenter.rb

```diff
@@ -13,6 +13,10 @@ def beis_identifier
     super.to_s
   end
 
+  def is_oda
+    (super) ? "ODA" : "Non-ODA"
```

Comment:
Does `super` need to be in brackets? I took them out and the tests passed

Reply:
I didn't have this, then Standard complained at me when I pushed to CI, although my VS Code complained locally... I'll double check this

Reply:
Yep, standard wants this (it's also what we've got further down in the file)

Reply:
🤔